CONTRIBUTING.md

```diff
@@ -441,10 +441,27 @@ The API reference is manually assembled in `doc/api/index.rst`.
 The *autodoc* sphinx extension will automatically create pages for each
 function/class/module listed there.
 
-You can reference classes, functions, and modules from anywhere (including docstrings)
-using <code>:func:\`package.module.function\`</code>,
-<code>:class:\`package.module.class\`</code>, or
-<code>:mod:\`package.module\`</code>.
+You can reference classes, functions, methods, and modules from anywhere 
```

Comment:
Change the order so that they match the order of the `<code>` examples.
```suggestion
You can reference functions, classes, methods, and modules from anywhere 
```